### PR TITLE
Probe performance

### PR DESCRIPTION
**Pull request by gmadip**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM python:2.7
+FROM python:3.6
 RUN pip install --upgrade pip
 ADD requirements.txt /tmp/requirements.txt
 RUN pip install -r /tmp/requirements.txt
 ADD . /code
 WORKDIR /code
-CMD [ "python", "./kafkaOffsetProbe.py" ]
+CMD [ "python", "./probe_new.py" ]
```
Comment (eljasala) [inline Dockerfile]: Why new file? And if it's intentional, why keep old one?

File: probe_new.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+# create the Gauge object to track the offset change
+registry = CollectorRegistry()
+producerOffsetTracker = Gauge('kafka_offset', 'the latest available offsets', ['service', 'topic', 'partition'],
+                              registry=registry)
+consumerGroupOffsetTracker = Gauge('cg_kafka_offset', 'the consumer group offsets',
+                                   ['service', 'topic', 'partition', 'consumergroup'], registry=registry)
+
+offset_info = {}
+kafka_hosts = "c1-26-usab1-kafka-1.oracle.callstats.io:9092"
```
Comment (eljasala) [inline probe_new.py]: This should be read from environment

File: probe_new.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    # Wait until both tasks are completed (should take
+    # around 2 seconds.)
+
+    # for task in tasks:
+    #    await task
+    # await task1
+    # await task2
+    # await task3
+
+    print(f"finished at {time.strftime('%X')}")
```
Comment (eljasala) [inline probe_new.py]: Is this needed?

File: probe_new.py [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+
+@app.route('/internal/metrics')
+def getKafkaOffset():
+    # fetch the latest available offsets from kafka and return the metrics
+    kafka_hosts = "c1-26-usab1-kafka-1.oracle.callstats.io:9092"
```
Comment (eljasala) [inline probe_new.py]: Hardcoded

File: probe_new.py (hunk 2 above)
Comment (gmadip) [reply probe_new.py]: probe_new is not required anymore. Will delete the file. It was kept for testing purpose.

File: probe_new.py (hunk 3 above)
Comment (gmadip) [reply probe_new.py]: probe_new is not required anymore. Will delete the file. It was kept for testing purpose.

File: probe_new.py (hunk 4 above)
Comment (gmadip) [reply probe_new.py]: probe_new is not required anymore. Will delete the file. It was kept for testing purpose.

File: Dockerfile (hunk 1 above)
Comment (gmadip) [reply Dockerfile]: Yes. I will change the docker file and push changes.

**Pull request closed without merging**